__tests__/data/fb_custom_audience.json

```diff
@@ -1,6 +1,730 @@
+              "ST",
+              "COUNTRY"
+            ],
+            "isHashRequired": false,
+            "disableFormat": false,
+            "audienceId": "aud1",
+            "isRaw": true,
+            "type": "UNKNOWN",
+            "subType": "ANYTHING",
+            "maxUserCount": "50"
+          },
+          "Enabled": true,
+          "Transformations": [],
+          "IsProcessorEnabled": true
+        },
+        "libraries": [],
+        "request": {
+          "query": {}
+        }
+      },
+      "output": {
+          "error": "Message Type is not present. Aborting message."
+        }
+  },
+  {
+      "description": "Unsupported Event type ",
+      "input": {
+        "message":{
+          "userId": "user 1",
+          "anonymousId": "anon-id-new",
+          "type":"Track",
+          "event": "event1",
+          "properties": {
+            "listData": {
+              "add": [
+                {
+                  "EMAIL": "shrouti@abc.com",
+                  "DOBM": "2",
+                  "DOBD": "13",
+                  "DOBY": "2013",
+                  "PHONE": "@09432457768",
+                  "GEN": "f",
+                  "FI": "Ms.",
+                  "MADID": "ABC",
+                  "ZIP": "ZIP ",
+                  "ST": "123abc ",
+                  "COUNTRY": "IN"
+                }
+              ],
+              "remove": [
+                {
+                  "EMAIL": "shrouti@abc.com",
+                  "DOBM": "2",
+                  "DOBD": "13",
+                  "DOBY": "2013",
+                  "PHONE": "@09432457768",
+                  "GEN": "f",
+                  "FI": "Ms.",
+                  "MADID": "ABC",
+                  "ZIP": "ZIP ",
+                  "ST": "123abc ",
+                  "COUNTRY": "IN"
+                }
+              ]
+            }
+          },
+          "context": {
+            "ip": "14.5.67.21",
+            "library": {
+              "name": "http"
+            }
+          },
+          "timestamp": "2020-02-02T00:23:09.544Z"
+        }, 
+        "destination": {
+          "Config": {
+            "accessToken": "ABC",
+            "userSchema": [
+              "EMAIL",
+              "DOBM",
+              "DOBD",
+              "DOBY",
+              "PHONE",
+              "GEN",
+              "FI",
+              "MADID",
+              "ZIP",
+              "ST",
+              "COUNTRY"
+            ],
+            "isHashRequired": false,
+            "disableFormat": false,
+            "audienceId": "aud1",
+            "isRaw": true,
+            "type": "UNKNOWN",
+            "subType": "ANYTHING",
+            "maxUserCount": "50"
+          },
+          "Enabled": true,
+          "Transformations": [],
+          "IsProcessorEnabled": true
+        },
+        "libraries": [],
+        "request": {
+          "query": {}
+        }
+      },
+      "output": {
+          "error": " NAN call is not supported "
```

Comment:
This test case is failing, Please change it to `Track call is not supported`

Reply:
Yeah. Updated